tests/nodeos_forked_chain_test.py

```diff
@@ -417,19 +423,23 @@ def getBlock(self, blockNum):
         if not prodChanged:
             if preKillBlockProducer!=blockProducer0:
                 prodChanged=True
+                Print("prodChanged = True")
```

Comment:
Are we keeping these print statements for diagnosing more failures over time?

Reply:
yes. it is hard to understand what is happening without those